### ISSUE TITLE
Should "Toggle all apps" be active if not all apps are selected above?

### ISSUE THREAD

**Issue by leonexcc** [KIProtect/klaro#117]
**The problem**

Currently the "Toggle all apps" is inactive if no app is active above. If one app is active it is also active. I find this a bit strange since it is called a toggle of all apps. The way it works today, makes it is obviously easier to deactivate all apps with one click. To be completely correct it should probably should have three states (none, some and all)

**Proposed solution**
 
* Make the behavior configurable (some active apps will change the toggle to active or not active)

**Alternative solutions**

* Remove the slider and add two buttons: "Allow all apps" and "Disallow all apps"

**Comment by jaller94** [comment]
I agree that the expected behaviour is a bit ambiguous. The reasoning was to favour opt-in, while keeping a the visual design of switches.

I oppose the introduction of a new configuration. The current configuration set is already complex, both for users of the library and the developers who test Klaro.

The alternative solution of two buttons seems a reasonable solution to avoid a new configuration while removing the ambiguity of what a click will do.

**Comment by leonexcc** [comment]
Btw. it looks like Orejime is doing exactly this:

![Bildschirmfoto 2019-11-21 um 16 07 42](https://user-images.githubusercontent.com/7428636/69350041-20daae80-0c79-11ea-9ab4-abb6e09c9f2c.png)

**Comment by jaller94** [comment]
Yeah.. they did good work and I hope Klaro will receive most changes they have made.

Off topic: Their entire idea to no rely on colour alone is pretty awesome.
![Screenshot_2019-11-22_20-23-47](https://user-images.githubusercontent.com/10872136/69454291-24982f00-0d66-11ea-8f08-b662b585093c.png)
☝ After just a second I am unsure which is "enabled" and which "disabled".

**Comment by leonexcc** [comment]
A simple checkbox would fix this to :-)

**Comment by balduin-kowalski** [comment]
A simple function to just hide the toggle in the config-file would also be nice, or is there one I did not find? When there are only 2-3 apps, a toggle-all is not really needed...

**Comment by lupo72** [comment]
I am not sure if the above is really addressing my problem - which is the following (using the example config) and a 

only mandatory choice is checked - when any additional apps are accepted the first choice always toggles the "allow all apps" button - which doesn't feel right to me, since i may have more than two options to allow or deny. 

![grafik](https://user-images.githubusercontent.com/1905736/85729018-ba2edf00-b6f8-11ea-9431-fedec4f02d7b.png)
